mfr/extensions/md/render.py

```diff
@@ -23,12 +16,11 @@ class MdRenderer(extension.BaseRenderer):
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
-        self.metrics.add('markdown_version', markdown.version)
 
     def render(self):
         """Render a markdown file to html."""
         with open(self.file_path, 'r') as fp:
-            body = markdown.markdown(fp.read(), extensions=[EscapeHtml()])
+            body = bleach.clean(fp.read(), **md_settings.BLEACH_WHITELIST)
```

Comment:
Although MFR uses `bleach` the same way OSF does, the library is not exactly the same due to python version difference.

Reply:
Not sure if it got noted somewhere else, but more specifically it causes problems with unclosed tags.

```
Mark down is great!
<p> okay </p

other stuff
more stuff

```
will usually just cut off the last 2 lines and end at at the 'okay' in a completed `<p>` tag

Reply:
I see. My test didn't have the extra lines which made me believe that python35 closes unclosed tags correctly. I will test this again. Thanks for the follow up @AddisonSchiller

Reply:
I can't remember the exact behavior, but I do recall it working correctly sometimes with unclosed tags, and other times displaying the above behavior. May need a bit more investigation to figure out the full extent of it. Bleach also had some github issues that explained the behavior as well. They may be linked on an old comment somewhere, or on the old PR. May be worth checking those out.